packages/xpath/src/context/Context.ts

```diff
@@ -37,5 +37,5 @@ export interface Context<T extends XPathNode> {
 
 	readonly functions: FunctionLibraryCollection;
 	readonly namespaceResolver: NamespaceResolver<T>;
-	readonly timeZone: Temporal.TimeZone;
+	readonly timeZone: Temporal.TimeZoneLike;
```

Comment:
Temporal defines `TimeZoneLike` like this:

```js
  /**
   * Any of these types can be passed to Temporal methods instead of a time zone ID.
   */
  type TimeZoneLike = string | ZonedDateTime;
```

In case this is useful, [ZonedDateTime definition](https://github.com/fullcalendar/temporal-polyfill/blob/bc1baf3875392ecd8522d40e7eecb55fa582808c/packages/temporal-spec/index.d.ts#L979).